packages/ses/src/lockdown.js

```diff
@@ -64,14 +66,15 @@ let priorRepairIntrinsics;
 /** @type {Error=} */
 let priorHardenIntrinsics;
 
+// @ts-expect-error harden always defined
+if (globalThis.harden) {
+  deleteProperty(globalThis, 'harden');
```

Comment:
Hmm, what happens with captured references to the pre-lockdown `harden`?

Reply:
Yeah, we need to figure out the story for this. Right now I'm simply removing any harden found, like in the case of our xsnap-worker.